x/bank/client/cli/cli_test.go

```diff
@@ -62,11 +64,14 @@ func (s *IntegrationTestSuite) TestGetBalancesCmd() {
 				fmt.Sprintf("--%s=1", flags.FlagHeight),
 			},
 			false,
-			&sdk.Coins{},
-			sdk.NewCoins(
-				sdk.NewCoin(fmt.Sprintf("%stoken", val.Moniker), s.cfg.AccountTokens),
-				sdk.NewCoin(s.cfg.BondDenom, s.cfg.StakingTokens.Sub(s.cfg.BondedTokens)),
-			),
+			&types.QueryAllBalancesResponse{},
+			&types.QueryAllBalancesResponse{
+				Balances: sdk.NewCoins(
+					sdk.NewCoin(fmt.Sprintf("%stoken", val.Moniker), s.cfg.AccountTokens),
+					sdk.NewCoin(s.cfg.BondDenom, s.cfg.StakingTokens.Sub(s.cfg.BondedTokens)),
+				),
+				Pagination: &query.PageResponse{},
+			},
```

Comment:
Could we add a test case with a non-empty query.PageResponse? Just to make sure all this pagination stuff actually works.

Reply:
Sure, will address this in #6973